landoapi/models/landing_job.py

```diff
@@ -82,30 +82,11 @@ class LandingJob(Base):
     # ordered, the resulting column will have the same order
     # as the definition order of the enum. This can be relied
     # on for comparisons or things like queries with ORDER BY.
-    status = db.Column(
-        db.Enum(LandingJobStatus), nullable=False, default=LandingJobStatus.SUBMITTED
-    )
-
-    # JSON object mapping string revision id of the form "<int>" (used because
-    # json keys may not be integers) to integer diff id. This is used to
-    # record the diff id used with each revision and make searching for
-    # Transplants that match a set of revisions easy (such as those
-    # in a stack).
-    # e.g.
-    #     {
-    #         "1001": 1221,
-    #         "1002": 1246,
-    #         "1003": 1412
-    #     }
-    revision_to_diff_id = db.Column(JSONB, nullable=False)
-
-    # JSON array of string revision ids of the form "<int>" (used to match
-    # the string type of revision_to_diff_id keys) listing the order
-    # of the revisions in the request from most ancestral to most
-    # descendant.
-    # e.g.
-    #     ["1001", "1002", "1003"]
-    revision_order = db.Column(JSONB, nullable=False)
+    status = db.Column(db.Enum(LandingJobStatus), nullable=True, default=None)
```

Comment:
This is because a job is not submitted until all related revisions are added (see landoapi/api/transplants.py:403). Previously, we did not need to wait between creating the job and submitting the job.

Reply:
I will revisit this as well, may be able to do all of that in one transaction.

Reply:
Due to the way the landing jobs table is locked when a transplant is requested, this will be a non-trivial change (and possibly more complex). I don't see any reason to not leave this nullable, or if we really want we could add a `CREATED` state, though alembic unfortunately doesn't play nice with postgres type migrations. I am inclined to leave this as-is.